dnf/repo.py

```diff
@@ -447,7 +447,8 @@ def __init__(self, remote_location, conf, handle, progress):
         self.remote_size = 0
         self.handle = handle
         self.conf = conf
-        s = self.conf.releasever or "" + self.conf.substitutions.get('basearch')
+        s = self.conf.releasever if self.conf.releasever else ''
```

Comment:
`"{}{}".format(self.conf.releasever or "", self.conf.substitutions.get("basearch"))` ?

Reply:
Adding brackets is simpler. `s = (self.conf.releasever or "") + self.conf.substitutions.get('basearch')`. But I don't like using logical operator 'or' with strings. So I rewrote code to two lines.

Reply:
OK. In Python documentation is written: The expression x or y first evaluates x; if x is true, its value is returned; otherwise, y is evaluated and the resulting value is returned.
So I will rewrote code to the simplest version with brackets.